simulator/src/data/project.js

```diff
@@ -75,7 +75,11 @@ export function projectSavedSet(param) {
  * @category data
  */
 export function saveOffline() {
-    const data = generateSaveData();
+    // Asking user name before starting the process of save
+    const projectNameByUser = prompt('Enter Project Name:');
```

Comment:
why ask for projectName in starting if it is already defined? means we should not ask for projectName if `getProjectName()` has some value( != `Untitled`).